package.json

```diff
@@ -5,13 +5,14 @@
   "main": "src",
   "scripts": {
     "prettier": "prettier \"{.,{src,tests_config}/**}/*.{js{,on},md}\" \"tests/**/jsfmt.spec.js\"",
+    "eslint": "eslint src/**",
```

Comment:
I would keep all the scripts @j-f1 what do you think?

Reply:
Yeah let’s keep these, but can you add `npm run eslint` to the `lint` script and `npm run eslint -- --fix` to the `lint:fix` script?